Jenkinsfile

```diff
@@ -31,39 +28,34 @@ node {
-                ADDONS_STATUS_CODE = sh(script: "curl -T ${addons_chartName} -u${BINTRAY_USERNAME}:${BINTRAY_APIKEY} https://api.bintray.com/content/pegasystems/helm-test-automation/helm-test-automation/${bintrayPackageVersion}/ --write-out '%{http_code}' ", returnStdout: true).trim()
-                UPDATE_STATUS_CODE = sh(script: "curl -T index.yaml -u${BINTRAY_USERNAME}:${BINTRAY_APIKEY} https://api.bintray.com/content/pegasystems/helm-test-automation/helm-test-automation/${bintrayPackageVersion}/ --write-out '%{http_code}'", returnStdout: true).trim()
-                PUBLISH_STATUS_CODE = sh(script: "curl -X POST -u${BINTRAY_USERNAME}:${BINTRAY_APIKEY} https://api.bintray.com/content/pegasystems/helm-test-automation/helm-test-automation/${bintrayPackageVersion}/publish --write-out '%{http_code}'", returnStdout: true).trim()
-                echo "DELETE_STATUS_CODE-- ${DELETE_STATUS_CODE}"
-                echo "PEGA_STATUS_CODE-- ${PEGA_STATUS_CODE}"
-                echo "ADDONS_STATUS_CODE-- ${ADDONS_STATUS_CODE}"
-                echo "UPDATE_STATUS_CODE-- ${UPDATE_STATUS_CODE}"
-                echo "PUBLISH_STATUS_CODE-- ${PUBLISH_STATUS_CODE}"
+            // Publish helm charts to test-automation GitHub Pages
+            withCredentials([usernamePassword(credentialsId: "helmautomation",
+              passwordVariable: 'AUTOMATION_APIKEY', usernameVariable: 'AUTOMATION_USERNAME')]) {
 
-                if ( "${DELETE_STATUS_CODE}" != "${curlSuccessStatus}"+"200" || "${PEGA_STATUS_CODE}" != "${curlSuccessStatus}"+"201" || "${ADDONS_STATUS_CODE}" != "${curlSuccessStatus}"+"201"
-                      || "${UPDATE_STATUS_CODE}" != "${curlSuccessStatus}"+"201" || "${PUBLISH_STATUS_CODE}" != '{"files":3}'+"200" ) {
-                    currentBuild.result = 'FAILURE'
-                    pullRequest.comment("Unable to publish helm charts to bintray repository. Please retry")
-                    error "This pipeline stops here! Unable to perform helm charts publish to bintray repository."
+                sh "git clone https://pegaautomationuser:${AUTOMATION_APIKEY}@github.com/pegaautomationuser/helmcharts.git --branch=gh-pages gh-pages"
+                sh "mv pega-${chartVersion}.tgz gh-pages/"
+                sh "mv addons-${chartVersion}.tgz gh-pages/"
+                sh "mv backingservices-${chartVersion}.tgz gh-pages/"
+                dir("gh-pages") {
+                  sh "helm repo index --merge index.yaml --url https://pegaautomationuser.github.io/helmcharts/ ."
+                  sh "cat index.yaml"    
+                  sh "git config user.email pegaautomationuser@gmail.com"
+                  sh "git config user.name ${AUTOMATION_USERNAME}"
+                  sh "git add ."
+                  sh "git commit -m \"Jenkins build to publish test artefacts of version ${chartVersion}\""
+                  sh "git push -u origin gh-pages"                                  
```

Comment:
Should we make it `git push --force`? if multiple builds are in progress and pushed test artifacts to the repository, it may lead to false positive push failures with above command. Alternatively, consider doing `git pull` before push.